Common/Statistics/KellyCriterionManager.cs

```diff
@@ -108,7 +108,7 @@ public void UpdateScores()
                         (1 / variance + 2 * averagePowered / variancePowered)
                         / _insightValues.Count - 1);
 
-                    KellyCriterionProbabilityValue =
+                    KellyCriterionProbabilityValue = kellyCriterionStandardDeviation.IsNaNOrZero() ? 1 :
```

Comment:
nit - before we were returning `0` for Nan and Zero (old `SafeDecimalCast`), think we should maintain existing behavior here?

Reply:
This change maintains existing behavior.  When `kellyCriterionStandardDeviation` is NaN or Zero, the CFD calculation is also NaN or Zero, and `KellyCriterionProbabilityValue` is 1.